src/components/questions-test/QuestionsTest.test.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import QuestionsTest from "./QuestionsTest";
+
+const question = {
+  question: "Question 1",
+  answers: [
+    {
+      answer: "Answer 1",
+      right: false,
+    },
+    {
+      answer: "Answer 2",
+      right: true,
+    },
+  ],
+};
+
+test("should display the first question with answer buttons", () => {
+  render(<QuestionsTest title="Questions" questions={[question]} />);
+
+  expect(screen.getByText(/questions/i)).toBeInTheDocument();
+  expect(screen.getByText(/question 1/i)).toBeInTheDocument();
+
+  expect(screen.getByRole("button", { name: /answer 1/i })).toBeInTheDocument();
+  expect(screen.getByRole("button", { name: /answer 2/i })).toBeInTheDocument();
+});
+
+test("should call onAnswerClick after answering", async () => {
+  const onAnswerClickMock = jest.fn();
+
+  render(
+    <QuestionsTest
+      title="Questions"
+      questions={[question]}
+      onQuestionAnswer={onAnswerClickMock}
+    />
+  );
+
+  userEvent.click(screen.getByRole("button", { name: /answer 2/i }));
+
+  expect(onAnswerClickMock).toHaveBeenCalled();
```

Comment:
In this test rather than asserting just for it invocation, you can also assert for number of times it's being called that could give us idea that this component is being rendered once.

Also you can assert the value that's being passed with this mock method using `toHaveBeenCalledWith()`

Reply:
I know others dislike testing the `onX` callbacks with mocks and they they should rather be tested by testing the parent component that actually wires it up to something.  Perhaps just drop this test for now and we can revisit it later if it't not tests as part of `ValencesTest`?

Reply:
Given the time since this was put up and the comments made, I'm going to go ahead and remove it and merge the other tests.  Feel free to revert my commit and open a new PR if you there are any issues with this.